src/Services/User.services.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const { generateUniqueID } = require('../utils/IdGenerator');
+
+let users = [];
```

Comment:
Why do we need let in this case?
Don`t see any reason why we don't use const for this array.
Try rewriting using const. We avoid not needed assignments.

Reply:
it was an easy way